go-controller/pkg/ovn/egressfirewall/egressfirewall.go

```diff
@@ -39,6 +48,13 @@ type destination struct {
 	cidrSelector string
 }
 
+func NewEgressFirewallPolicies(kubeInterface kube.Interface) *EgressFirewallPolicies {
+	return &EgressFirewallPolicies{
+		egressFirewalls: make(map[string]*egressFirewall),
```

Comment:
you receive the kubeInterface as an argument but you don't pass it to the object